### PR TITLE
Fixed: (CinemaZ, ExoticaZ, PrivateHD) Better Indexer Response Log Cleansing

### LINKED ISSUE #939: Cleanse ExoticaZ and sister sites response in logs
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

2022-04-07 02:05:26.1|Error|ExoticaZ|An error occurred while processing indexer feed. https://exoticaz.to/api/v1/jackett/torrents?in=1&type=0&search=SSIS-356

### Expected Behavior

Get results from searching exoticaz

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS: 
- Prowlarr:
- Docker Install:
- Using Reverse Proxy:
- Browser:
```

### What branch are you running?

Master

### Trace Logs?

https://0bin.net/paste/fYUx5Uzx#-jwH95oIvphJ41Sm4Rmd34yRcXdQItvghMzi0QjNbbK

[AB#2936](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2936)

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- Tests - N/A - add additional sites to regex
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #939

### CODE REVIEW

**Comment by bakerboy448** [comment]
Regex Proof
image

File: src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ public class CleanseLogMessage
                 new Regex(@"(?<=\?|&)(X-Plex-Client-Identifier|X-Plex-Token)=(?<secret>[^&=]+?)(?= |&|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
 
                 // Indexer Responses
-                new Regex(@"avistaz\.[a-z]{2,3}\\\/rss\\\/download\\\/(?<secret>[^&=]+?)\\\/(?<secret>[^&=]+?)\.torrent", RegexOptions.Compiled | RegexOptions.IgnoreCase),
+                new Regex(@"(?:avistaz|exoticaz|cinemaz)\.[a-z]{2,3}\\\/rss\\\/download\\\/(?<secret>[^&=]+?)\\\/(?<secret>[^&=]+?)\.torrent", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```
Comment (Qstick) [inline src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs:54]: There are more I thought

Comment (bakerboy448) [reply src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs:54]: That's the format known for 2/3 and assumed for the 3rd.

What else is using that format?

Comment (Qstick) [reply src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs:54]: PHD is also avistaz based, so assume they are?

Comment (bakerboy448) [inline src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs:54]:
```suggestion
                new Regex(@"(?:avistaz|exoticaz|cinemaz|privatehd)\.[a-z]{2,3}\\\/rss\\\/download\\\/(?<secret>[^&=]+?)\\\/(?<secret>[^&=]+?)\.torrent", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```

**Review by Qstick — CHANGES_REQUESTED** [review]
Squash

**Pull request merged**